tests/tools/oesign/test-inputs/CMakeLists.txt

```diff
@@ -56,14 +56,32 @@ add_custom_command(
   COMMAND python ${CMAKE_CURRENT_SOURCE_DIR}/make-oesign-config.py
           --config_file new_security_version.conf --security_version 2)
 
+add_custom_command(
+  OUTPUT invalid_debug.conf
```

Comment:
Consider naming the configurations so that it is clear what the invalid structure being tested is, e.g. `debug_out_of_range.conf`, `lowercase_debug_property.conf`, `duplicate_num_heap_pages.conf`,

Reply:
renamed the config files